M2/Macaulay2/packages/QuadraticIdealExamplesByRoos.m2

```diff
@@ -0,0 +1,417 @@
+newPackage(
+	"QuadraticIdealExamplesByRoos",
+	Version => "0.1",
+	Date => "June, 2023",
+	AuxiliaryFiles => false,
+	Authors => {{Name => "David Eisenbud", Email => "de@msri.org"},
+	    {Name => "Michael Perlman", Email => "mperlman@umn.edu"}, 
+	    {Name => "Ritvik Ramkumar", Email => "ritvikr@cornell.edu"},
+	    {Name => "Deepak Sireeshan", Email => "dsbx7@umsystem.edu"},
+	    {Name => "Aleksandra Sobieska", Email => "asobieska@wisc.edu"},
+	    {Name => "Teresa Yu", Email => "twyu@umich.edu"},
+	    {Name => "Jacob Zoromski", Email => "jzoromsk@nd.edu"} },
+	Headline => "Examples of Quadratic Ideals with Embedding Dimension Four by Jan-Erik Roos",
+	PackageExports => {"Depth"},
+	DebuggingMode => true)
+export {
+ "roosTable", 
+ "higherDepthTable", 
+ "depthZeroTable",
+ "almostKoszul",
+ "roosIsotopes",
+ "onedimToricIrrationalPoincare",
+ "twodimToricIrrationalPoincare"
+}
+
+
+---TO DO: make this a method to allow user to pick coefficient field
+roosTable = (S = QQ[x,y,z,u];
```

Comment:
If someone knows better correct me if I'm wrong, but my understanding is that defining global rings when the package loads can be problematic.

Reply:
I need to look, but that is probably why the package installation is failing now?